services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -551,59 +557,52 @@ async def _remove_containers_save_state_and_outputs() -> None:
             dynamic_sidecar_client: DynamicSidecarClient = get_dynamic_sidecar_client(
                 app
             )
+            dynamic_sidecar_endpoint: AnyHttpUrl = (
+                scheduler_data.dynamic_sidecar.endpoint
+            )
+
             try:
-                await dynamic_sidecar_client.begin_service_destruction(
-                    dynamic_sidecar_endpoint=scheduler_data.dynamic_sidecar.endpoint
-                )
-            except BaseClientHTTPError as e:
+                await dynamic_sidecar_client.remove_containers(dynamic_sidecar_endpoint)
+            except (BaseClientHTTPError, TaskClientResultError) as e:
                 logger.warning(
                     (
-                        "Could not contact dynamic-sidecar to begin destruction of "
+                        "There was an issue while removing contains for "
```

Comment:
I am a bit unsure of this warning to be honnest.
If I get that correctly, there was an unexpected error while stopping the service right? then you say it will continue with service removal. Is that really the case? I thought then they were remaining as zombies.

Reply:
This has little consequence. Usually it happens because the containers are gone and you want to remove them.
I would still leave it as a warning and maybe have some better text around it. I've tried to word it better.